### DISCUSSION TITLE
Best Theme for VS Code in 2022

### DISCUSSION THREAD

**Discussion by J-Nokwal** [Schweinepriester/github-profile-achievements#496] [Q&A] [answered]
Best Theme for VS Code

**Comment by bawejagb** [comment, marked as answer]
I recommend you use the **JellyFish Theme** Extention, it has very great text fonts. ❤️‍🔥
Must Try, I've been using it for the past few months, and I love it.
Link:[https://marketplace.visualstudio.com/items?itemName=PawelBorkar.jellyfish](url)

**Comment by lambdakilo** [comment]
@J-Nokwal Please don't spam in GitHub Discussions. I've already filed a report of you but I wanted to also let you know personally this is not okay. ☹️

**Comment by Preetiraj3697** [comment]
I recommend you use the NightOwl Theme Extention, it has very great text fonts. ❤️‍🔥
Must Try, I've been using it for the past 1 year, and I love it.

**Comment by Wojtekb30** [comment]
I personally like the default dark one most.

**Comment by khatrisagar** [comment]
i am using dark ocean

**Comment by tom-mcIvor** [comment]
its got to be the vampire theme 💯

**Comment by thehassantahir** [comment]
> Best Theme for VS Code

Dracula Theme 🧛 https://draculatheme.com/
